### DISCUSSION TITLE
Can't bind to 'showCurrentPageReport' since it isn't a known property of 'p-table'

### DISCUSSION THREAD

**Discussion by CoachRice** [primefaces/community#695] [PrimeNG] [answered]
My table works fine if there is no [showCurrentPageReport]="true". As soon as I add it, I got error: Can't bind to 'showCurrentPageReport' since it isn't a known property of 'p-table'
Anyone knows why?

<p-table #lstDevices [value]="devices" [paginator]="true" [rows]="5" [showCurrentPageReport]="true"
            currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries"
             styleClass="p-datatable-striped" [scrollable]="true" scrollDirection="both" scrollHeight="200px"
             [globalFilterFields]="['Name', 'ID']">
...
</p-table>

**Comment by cetincakiroglu** [comment]
Could you please share a stackblitz example so we can identify the issue?

**Comment by CoachRice** [comment]
I realized that this is probably because primeng installed in my project is an old version 7.0.0. And 'showCurrentPageReport' is not supported in this version.
I tried to upgrade primeng to a newer version but failed. It gave me a lot of compatibility errors. I will check if I can upgrade primeng and test it again. Thanks for the reply.

**Reply by CoachRice** [reply, marked as answer]
After upgrading to PrimeNG 17, problem solved.
